templates/index.html

```diff
@@ -1,129 +1,181 @@
+					This is some stuff about music in the US.
+					This is some stuff about music in the US.
+					This is some stuff about music in the US.
+					This is some stuff about music in the US.
+					This is some stuff about music in the US.
+					This is some stuff about music in the US.
+					This is some stuff about music in the US.
+					This is some stuff about music in the US.
+					This is some stuff about music in the US.
+					This is some stuff about music in the US.
+				</small>
+			</div>
+		</div>
+		<div class = "col-md-9" id = "mainPanel">
+			<!-- loading icons here -->
+			<div class = "row" id = "navbar">
+			<div class="col-md-2">
+				<div class = 'icon' id='pop' >
+					<INPUT type="image" src="{{url_for('static',filename = 'images/pop.png')}}"
+					onclick="updateData()" width="85%">
+				</div>
+			</div>
+
+			<div class="col-md-2">
+				<div class = 'icon' id='indie' href=#>
+					<INPUT type="image" src="{{url_for('static',filename = 'images/indie.png')}}"
+					onclick="updateData()" width="85%">
+				</div>
+			</div>
+
+			<div class="col-md-2">
+				<div class='icon' id='country' href=#>
+					<INPUT type="image" src="{{url_for('static',filename = 'images/country.png')}}"
+					value="" onclick="updateData()" width="85%">
+				</div>
+			</div>
+
+			<div class="col-md-2">
+				<div class='icon' id='classic' href=#>
+					<INPUT type="image" src="{{url_for('static',filename = 'images/classic.png')}}"
+					onclick="updateData()" width="85%">
+				</div>
+			</div>
+
+			<div class="col-md-2">
+				<div class='icon' id='hiphop' href=#>
+					<INPUT type="image" src="{{url_for('static',filename = 'images/hiphop.png')}}"
+					onclick="updateData()" width="85%">
+				</div>
+			</div>
+
+			<div class="col-md-2">
+				<div class='icon' id='metal' href=#>
+					<INPUT type="image" src="{{url_for('static',filename = 'images/metal.png')}}"
+					onclick="updateData()" width="85%">
+				</div>
+			</div>
+
+		</div>
+		<div class= "row">
+		<div class= "col-md-11" id="mapCanvas">
+			<div> <!-- div and svg to hold the map. -->
+				<svg id = "statesvg" viewBox = "0 0 960 630" preserveAspectRatio="xMidYMid meet"></svg>
+			</div>
+			<div id="tooltip"></div><!-- div to hold tooltip. -->
+		</div>
+		<div class = col-md-1 id="legend"></div><!-- div to hold legend. -->
+		</div>
 
 </div>
 </body>
 
 
-<!-- SOURCE: https://gist.github.com/NPashaP/a74faf20b492ad377312 -->
-<script src="{{url_for('static',filename = 'uStates.js')}}"></script> <!-- creates uStates. -->
-<script src="http://d3js.org/d3.v3.min.js"></script>
+<script src = "https://cdnjs.cloudflare.com/ajax/libs/d3/4.2.2/d3.min.js"></script><!-- D3 graphing library -->
+<script src="https://d3js.org/d3-scale-chromatic.v1.min.js"></script>
+<script src = "https://d3js.org/topojson.v1.min.js"></script><!-- topojson library -->
+<script src="{{url_for('static',filename = 'uStates.js')}}"></script><!-- creates uStates. -->
 <script>
```

Comment:
[RC]
Move to own `js` file.

Reply:
Sorry I'm not sure how to do this. I'm sure it's very simple but I can't find anything online...probably because I'm google-ing it incorrectly.

Reply:
Same as uStates.js in the line above (128). 
But let's make it a SC for now and I'll incorporate it after we merge this one.